lib/plenario_aot/aot_worker.ex

```diff
@@ -28,7 +28,7 @@ defmodule PlenarioAot.AotWorker do
   # SERVER IMPLEMENTATION
 
   def handle_call({:process, meta}, _, state) do
-    path = "/tmp/#{meta.slug}.json"
+    {:ok, path} = Briefly.create()
```

Comment:
I'm admittedly ignorant, but the Briefly magic does lose the ability to restart a failed ingest without redownloading the source. Is that something we care about?

Reply:
Good point, but the ingest as it exists isn't smart enough to look for an existing file. So no functionality lost. ;)

Reply:
The ingest, as I rewrote it in #325, removes any file that matches the name at the start of the process to ensure a clean source file.

Reply:
Great, not a concern then